tests/components/nextcloud/test_update.py

```diff
@@ -0,0 +1,59 @@
+"""Tests for the Nextcloud init."""
+
+from copy import deepcopy
+from unittest.mock import patch
+
+from syrupy.assertion import SnapshotAssertion
+
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+from . import init_integration
+from .const import NC_DATA, VALID_CONFIG
+
+
+async def test_setup_entity_without_update(
+    hass: HomeAssistant, snapshot: SnapshotAssertion
+) -> None:
+    """Test update entity is created w/o available update."""
+    with patch("homeassistant.components.nextcloud.PLATFORMS", [Platform.UPDATE]):
+        entry = await init_integration(hass, VALID_CONFIG, NC_DATA)
+
+    assert entry.state == ConfigEntryState.LOADED
```

Comment:
I notice you always do these together, you can also do the `assert hass.config_entries.async_setup()`, it will make the extra assert obsolete